### PR TITLE
add oauth

### PR DESCRIPTION
**Pull request by vdvoretskyi**

### CODE REVIEW

File: src/main/java/com/km220/dao/station/StationRepositoryV2.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.km220.dao.station;
+
+import java.util.UUID;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface StationRepositoryV2 extends CrudRepository<StationEntity, UUID> {
+
+  StationEntity findByNumber(String number);
```
Comment (maxpavlovdp) [inline src/main/java/com/km220/dao/station/StationRepositoryV2.java:10]: this what I wanted

**Review by maxpavlovdp — CHANGES_REQUESTED** [review]

**Comment by maxpavlovdp** [comment]
1. monobank callback faces 401 I believe
2. 401 erro is used in ewelink and in our logic. Is it ok?
image
image

**Comment by maxpavlovdp** [comment]
user km220.com@gmail.com should see only station 1

File: src-js/react/src/App/adminPage/AdminPage.tsx [hunk 2]
```diff
@@ -2,16 +2,25 @@ import React, { useEffect, useState } from "react";
 import { Col, Container, Row } from "react-bootstrap";
 import { useTranslation } from "react-i18next";
 import DropDown from "../../components/dropDown/DropDown";
+import ErrorPage from "../../components/error-page/ErrorPage";
+import Login from "../../components/loginPage/Login";
+import UserInfo from "../../components/userInfo/UserInfo";
 import { useAppDispatch, useAppSelector } from "../../hooks/reduxHooks";
 import { getAdminStatistics } from "../../store/reducers/ActionCreators";
 import "./adminPage.css";
```
[the remark this answers is not in the document]
Comment (dimaborbotko) [reply src-js/react/src/App/adminPage/AdminPage.tsx]: renamed

**Comment by maxpavlovdp** [comment]
http://49.12.19.42:8080/swagger-ui/index.html#/order-controller/callBackMono

**Pull request merged**